apps/data-pipeline/larc-scraper/src/lib.ts

```diff
@@ -2,15 +2,25 @@ import type { database } from "@packages/db";
 import { and, eq } from "@packages/db/drizzle";
 import type { Term } from "@packages/db/schema";
 import { larcSection, websocCourse } from "@packages/db/schema";
-import { sleep } from "@packages/stdlib";
+import { parseStartAndEndTimes, sleep } from "@packages/stdlib";
 import { load } from "cheerio";
 import { fetch } from "cross-fetch";
+import { parseMeetingDays } from "../../../api/src/schema/lib/day.ts";
```

Comment:
Should probably remove this and import from stdlib?

Reply:
Ahh can't believe I missed this :facepalm: . Good catch